src/software/thunderscope/thunderscope.py

```diff
@@ -68,6 +68,66 @@
 REFRESH_INTERVAL_MS = 5
 GAME_CONTROLLER_URL = "http://localhost:8081"
 
+from pyqtgraph.dockarea.Dock import DockLabel
+
+
+def updateStylePatched(self):
```

Comment:
can you move this function outside of this file, put it in `dock_label_style.py` in thunderscope/ and import it here

also you can't really tell the difference between a tab and a dock (you can't drag tabs but you can drag docks which the current style differentiates pretty well imo and I personally prefer that). if you want it to be blue, can't you just change the purple to blue?

 the text looks kind of squashed, can you increase the spacing a bit between the text and the border line?